lib/rules/v-bind-style.js

```diff
@@ -17,6 +17,7 @@ const utils = require('../utils')
 
 module.exports = {
   meta: {
+    type: 'layout',
```

Comment:
I feel this is `suggestion` because this is about neither whitespaces nor punctuators.

Reply:
`all the parts of the program that determine how the code looks rather than how it executes. These rules work on parts of the code that aren’t specified in the AST`